packages/components/src/components/button/button.scss

```diff
@@ -22,6 +22,9 @@
      */
     text-transform: none !important;
 
+    > * {
+        pointer-events: none;
+    }
```

Comment:
What's the reason this is needed? It's a bit vague to select all subclasses and nullify pointer events.
Assuming you just want to prevent `highlighting` of text, you can use `user-select: none`. [doc](https://developer.mozilla.org/en-US/docs/Web/CSS/user-select)

Reply:
It was done so pointer events aren't emitted from `.btn`'s children which will affect click tracking in GTM. I don't think there's any situation where one of its children needs their own events so it should be safe to go forward with this. With regards to BEM linter, let me update.